wcfsetup/install/files/js/WoltLabSuite/Core/Ui/ItemList.js

```diff
@@ -110,7 +110,7 @@ define(['Core', 'Dictionary', 'Language', 'Dom/Traverse', 'EventKey', 'WoltLabSu
 							for (var i = 0, length = values.length; i < length; i++) {
 								input = elCreate('input');
 								input.type = 'hidden';
-								input.name = options.submitFieldName.replace(/{$objectId}/, values[i].objectId);
+								input.name = options.submitFieldName.replace(/\{\$objectId\}/, values[i].objectId);
```

Comment:
```suggestion
								input.name = options.submitFieldName.replace('{$objectId}', values[i].objectId);
```

Reply:
Seeing this: Is there a need to perform multiple replacements? Then we'll need a regular expression + the `/g` modifier.

Reply:
No, it is fine, this is a single `name` attribute.